src/Bicep.Core/Registry/Oci/OciArtifactResult.cs

```diff
@@ -14,7 +14,7 @@ namespace Bicep.Core.Registry.Oci
 {
     public class OciArtifactResult
     {
-        public OciArtifactResult(BinaryData manifestBits, string manifestDigest, ImmutableDictionary<string, BinaryData> layers)
+        public OciArtifactResult(BinaryData manifestBits, string manifestDigest, ImmutableList<(string MediaType, BinaryData Data)> layers)
```

Comment:
Should we use `ImmutableArray` here instead? `ImmutableList` is more suitable if the layers will be updated frequently.

Reply:
57a9fca7c6072858c33d42ebd8aec05f79906348